src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncContext.Unix.cs

```diff
@@ -234,25 +342,42 @@ private abstract class SendOperation : WriteOperation
             public int Offset;
             public int Count;
 
+            public SendOperation(SocketAsyncContext context) : base(context) { }
+
             protected sealed override void Abort() { }
 
             public Action<int, byte[], int, SocketFlags, SocketError> Callback
             {
                 set => CallbackOrEvent = value;
             }
 
-            protected sealed override void InvokeCallback() =>
+            protected override void InvokeCallback() =>
```

Comment:
Should this be abstract?

Reply:
There are several derived types that don't override it.  Either it needs an implementation, or this implementation needs to be copied up to those types.

Reply:
Ok, makes sense to me.